src/Strategy/Ruby/Gemspec.hs

```diff
@@ -0,0 +1,108 @@
+module Strategy.Ruby.Gemspec (
+  rubyString,
+  parseRubyAssignment,
+  Assignment (..),
+  readAssignments,
+  parseRubyArray,
+  parseRubyWordsArray,
+  rubyLicenseValuesP,
+) where
+
+import Control.Applicative ((<|>))
+import Data.Char (isSpace)
+import Data.Functor (($>))
+import Data.List.Extra (singleton)
+import Data.String.Conversion (toText)
+import Data.Text (Text)
+import Data.Void (Void)
+import Text.Megaparsec (MonadParsec (eof), Parsec, anySingle, anySingleBut, between, choice, lookAhead, many, optional, sepBy, skipManyTill, takeWhile1P, takeWhileP, try)
+import Text.Megaparsec.Char (char, space, space1, string)
+
+type Parser = Parsec Void Text
+
+-- |Given a single start delimiter, return start/end delimiters
+selectDelim :: Char -> (Char, Char)
+selectDelim = \case
+  '{' -> ('{', '}')
+  '<' -> ('<', '>')
+  '(' -> ('(', ')')
+  '[' -> ('[', ']')
+  c -> (c, c)
+
+rubyString :: Parser Text
+rubyString =
+  -- '.freeze' is a ruby idiom that turns a string into an immutable version of
+  -- itself. In this case it's safe to just ignore it and take the string it's
+  -- attached to.
+  toText <$> (stringText <* optional (string ".freeze"))
```

Comment:
It's idiomatic to say `.freeze`, but `.freeze()` is also valid Ruby code. Should we deal with that too?

Reply:
This might be going too far, but we could probably just ignore anything after a `.` following a closing string delimiter.

This would deal with things like `license = 'mit'.upcase` or `license = '  Apache 2.0 '.trim`

I'd say we ignore this if you didn't see it when you downloaded 1000 gemspecs or whatever the number was.

Reply:
I'm going to mark this resolved per our earlier conversation.